projects/chaos/inc/chaos_events.h

```diff
@@ -2,18 +2,42 @@
 
 // High priority messages in the event queue
 typedef enum {
-  NUM_CHAOS_EVENTS_CRITICAL = 0,
-} ChaosEventCritical;
+  CHAOS_EVENT_RELAY_ERROR = 0,
+  NUM_CHAOS_EVENTS_CRITICAL,  // 1
+} ChaosEventsCritical;
 
-// CAN messages in the event queue
+// CAN related messages in the event queue
 typedef enum {
-  CHAOS_EVENT_CAN_UV_OV = NUM_CHAOS_EVENTS_CRITICAL + 1,
-  NUM_CHAOS_EVENTS_CAN,
-} ChaosEventCAN;
+  CHAOS_EVENT_CAN_TRANSMIT_ERROR = NUM_CHAOS_EVENTS_CRITICAL + 1,  // 2
+  CHAOS_EVENT_CAN_FAULT,
+  CHAOS_EVENT_CAN_RX,
+  CHAOS_EVENT_CAN_TX,
+  CHAOS_EVENT_RETRY_RELAY,
+  NUM_CHAOS_EVENTS_CAN,  // 7
+} ChaosEventsCAN;
 
-// State transition messages in the event queue
+// State transition messages in the event queue (only 1 should be in the queue at a time so order is
+// irrelevant).
 typedef enum {
-  NUM_CHAOS_EVENTS_FSM = NUM_CHAOS_EVENTS_CAN + 1,
-} ChaosEventFSM;
+  CHAOS_EVENT_NO_OP = NUM_CHAOS_EVENTS_CAN + 1,  // 8
+  CHAOS_EVENT_MONITOR_ENABLE,
+  CHAOS_EVENT_MONITOR_DISABLE,
+  CHAOS_EVENT_OPEN_RELAY,
+  CHAOS_EVENT_CLOSE_RELAY,
+  CHAOS_EVENT_RELAY_OPENED,
+  CHAOS_EVENT_RELAY_CLOSED,
+  CHAOS_EVENT_GPIO_IDLE,
+  CHAOS_EVENT_GPIO_CHARGE,
+  CHAOS_EVENT_GPIO_DRIVE,
+  NUM_CHAOS_EVENTS_FSM  // 18
+} ChaosEventsFSM;
+
+typedef enum {
+  CHAOS_EVENT_SEQUENCE_IDLE = NUM_CHAOS_EVENTS_FSM + 1,  // 19
+  CHAOS_EVENT_SEQUENCE_CHARGE,
+  CHAOS_EVENT_SEQUENCE_DRIVE,
+  CHAOS_EVENT_SEQUENCE_RESET,
+  NUM_CHAOS_EVENT_SEQUENCES,  // 23
+} ChaosEventSequence;
```

Comment:
What are these used for? Does `NUM_CHAOS_EVENTS` need to be updated?

Reply:
These are used for the event sequencer that schedules events for for different states to be used by the other FSMs.